tests/storage/migration.go

```diff
@@ -742,6 +742,18 @@ var _ = Describe(SIG("Volumes update with migration", decorators.RequiresTwoSche
 				libwait.WaitForVMIPhase(vmi, []v1.VirtualMachineInstancePhase{v1.Running})
 			})
 		})
+
+		It("should be possible to rollback a volume migration to the original volumes", func() {
```

Comment:
Is it possible to add a rollback step in one of the existing tests?

Reply:
yes, I can merged it with the DV to DV test